src/components/AddFood.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { Form } from 'react-bootstrap';
+
+function AddFood(props){
+    let formStyle = {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center'
+    }
+    return (
+        <Form onSubmit = {props.onAdd} style = {formStyle}>
```

Comment:
although this works, try to get used to working with react forms that change the state dynamically and that is the information in the inputs value. It is good practices for thinking in react. After that you can call the function with the inputs that are in the state.

Reply:
Hi Jorge!

I write you because I don't really know what's the meaning of 'react forms' in this feedback.

Sorry for the inconveniences,
Regards.

Reply:
Similar to how you did in the FoodBox component for the quantity input. I will slack you one solution where you will see how these are implemented.